pages/task/read/content/index.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   task: {
-    title: 'New task',
+    title: 'Review PIL Application',
```

Comment:
Just thinking, this content is all PIL specific but will need to change depending upon what type of task is being actioned.

When I created the ticket I was thinking we would put the structure in place to do that now, but I'm open to leaving it until we implement another task other than PIL endorsement?

Reply:
I'd have the logic exist for it to be able to change, but leave the other branches empty.

That is, have a _way_ to define dynamic content based on task type/state, but don't actually define any others.

Reply:
I would say yes.